views/components/tabComponent.njk

```diff
@@ -23,42 +27,64 @@
       }
     ]
   }) }}
+
+  Options:
+  - activeTab: Current active tab ID
+  - containerId: HTML ID for the tab container
+  - casesData: Data array to pass to partials (optional)
+  - sortOrder: Sort order to pass to partials (optional)
+  - tabs: Array of tab configuration objects:
+    * id: Unique tab identifier
+    * text: Display text for navigation
+    * href: URL for tab navigation
+    * partial: Path to Nunjucks partial template (optional)
+    * content: Direct HTML content (optional, alternative to partial)
+
+  Features:
```

Comment:
Observation: would be good to have `Features:` section above `Usage:` so we have consistency in these nunjuck macros.